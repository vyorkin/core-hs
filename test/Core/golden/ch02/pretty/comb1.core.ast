[(Name "B",[Name "x",Name "y",Name "z"],EAp (EVar (Name "x")) (EAp (EVar (Name "y")) (EVar (Name "z")))),(Name "C",[Name "x",Name "y",Name "z"],EAp (EAp (EVar (Name "x")) (EVar (Name "z"))) (EVar (Name "y"))),(Name "W",[Name "x",Name "y"],EAp (EAp (EVar (Name "x")) (EVar (Name "y"))) (EVar (Name "y")))]
