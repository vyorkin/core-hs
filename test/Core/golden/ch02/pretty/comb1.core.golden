B x y z = x (y z)
C x y z = (x z) y
W x y = (x y) y